app/controllers/katello/api/v2/repositories_controller.rb

```diff
@@ -130,6 +130,15 @@ def sync
       respond_for_async :resource => task
     end
 
+    api :POST, "/repositories/:id/export", N_("Export a repository")
+    param :id, :identifier, :required => true, :desc => N_("repository ID")
+    param :since, String, :desc => N_("ISO 8601 date of last export (ex: 2010-01-01T12:00:00), useful for exporting deltas.")
+    def export
+      Rails.logger.warn("Export called with date #{params[:since]}")
```

Comment:
Why a warning and not just an info message here? Also, it would be better to use the katello logger so we know where this message originated from -- http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Custom-Plugin-Loggers